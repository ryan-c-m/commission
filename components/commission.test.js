import 'react-native';
import React from 'react';
import Commission from './commission';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <Commission />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});

