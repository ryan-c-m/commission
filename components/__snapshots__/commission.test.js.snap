// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 30,
        "marginBottom": 20,
        "marginTop": 60,
      }
    }
  >
    Commissions
  </Text>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "backgroundColor": "#fff",
        "flex": 1,
        "flexDirection": "column",
        "minWidth": 300,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-evenly",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontFamily": "sans-serif",
            "fontSize": 25,
          }
        }
      >
        Sale price
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontFamily": "sans-serif",
            "fontSize": 25,
          }
        }
      >
        $
      </Text>
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onChangeText={[Function]}
        placeholder="0"
        style={
          Object {
            "fontSize": 30,
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-evenly",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontFamily": "sans-serif",
            "fontSize": 25,
          }
        }
      >
        Agent fee
      </Text>
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onChangeText={[Function]}
        placeholder="0"
        style={
          Object {
            "fontSize": 30,
          }
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontFamily": "sans-serif",
            "fontSize": 25,
          }
        }
      >
        %
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-evenly",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontFamily": "sans-serif",
            "fontSize": 25,
          }
        }
      >
        Advertising
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontFamily": "sans-serif",
            "fontSize": 25,
          }
        }
      >
        $
      </Text>
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onChangeText={[Function]}
        placeholder="0"
        style={
          Object {
            "fontSize": 30,
          }
        }
      />
    </View>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "marginBottom": 20,
        "marginTop": 60,
      }
    }
  >
    Commission: $
    0
  </Text>
</View>
`;
